index.bs

```diff
@@ -200,7 +201,7 @@ Gamepad {#gamepad-differences}
 The following {{Gamepad}} attributes MUST exhibit the following behavioral restrictions when the {{Gamepad}} has been returned by an {{XRInputSource}}'s {{XRInputSource/gamepad}} attribute.
 
   - {{XRInputSource/gamepad}}'s {{Gamepad/id}} attribute MUST be an empty string (<code>""</code>).
-  - {{XRInputSource/gamepad}}'s {{Gamepad/index}} attribute MUST be <code>-1</code> if it is not exposed via {{navigator.getGamepads()}}, otherwise it should be assigned as specified by [[GAMEPAD]].
+  - {{XRInputSource/gamepad}}'s {{Gamepad/index}} attribute MUST be <code>-1</code> if it is not exposed via {{navigator.getGamepads()}}, otherwise it should be assigned as specified by [=select an unused gamepad index=].
```

Comment:
```suggestion
  - {{XRInputSource/gamepad}}'s {{Gamepad/index}} attribute MUST be <code>-1</code> if it is not exposed via {{navigator.getGamepads()}}, otherwise it should be assigned as specified by [=select an unused gamepad index|selecting an unused gamepad index=].
```
```suggestion
  - {{XRInputSource/gamepad}}'s {{Gamepad/index}} attribute MUST be <code>-1</code> if it is not exposed via {{navigator.getGamepads()}}, otherwise it should be assigned as specified by [=select an unused gamepad index=].
```